db/migrate/20200916000828_add_active_to_sponsor_seller.rb

```diff
@@ -0,0 +1,5 @@
+class AddActiveToSponsorSeller < ActiveRecord::Migration[6.0]
+  def change
+    add_column :sponsor_sellers, :active, :boolean
```

Comment:
can you add default: true

Reply:
and nullable: false

Reply:
You can also do the migration here SponsorSellers.all.each { |s| s.update(active: true) }

Reply:
just realized this after making the pr, adding now!

Reply:
I think change_column_null :sponsor_sellers, :active, false, true also works, by setting nullable to false, and changing all null columns to true